crates/infra/cli/src/commands/test/mod.rs

```diff
@@ -44,11 +44,10 @@ fn test_cargo() -> Result<()> {
 
     Command::new("cargo")
         .args(["nextest", "run"])
+        .args(["--exclude", "solidity_testing_perf"]) // Requires callgrind, which may not be available
         .flag("--workspace")
         .flag("--all-features")
-        .flag("--lib")
-        .flag("--bins")
-        .flag("--examples")
+        .flag("--all-targets")
```

Comment:
I think you want to add `--tests` instead, which compiles the source files of each package as tests. The difference is that `--all-targets` also includes `--benches` which is what we want to exclude from normal testing (and why you need to exclude `solidity_testing_perf`).